rtbkit/plugins/augmentor_interface/http_augmentor_interface.cc

```diff
@@ -0,0 +1,486 @@
+#include "http_augmentor_interface.h"
+#include "jml/db/persistent.h"
+#include "soa/service/http_client.h"
+#include "soa/utils/generic_utils.h"
+#include "rtbkit/common/messages.h"
+#include "rtbkit/plugins/bid_request/openrtb_bid_request_parser.h"
+#include "rtbkit/openrtb/openrtb_parsing.h"
+#include "rtbkit/core/router/router.h"
+#include "jml/arch/futex.h"
+
+using namespace Datacratic;
+using namespace RTBKIT;
+
+namespace {
+    DefaultDescription<OpenRTB::BidRequest> desc;
+
+    std::string httpErrorString(HttpClientError code)  {
```

Comment:
I think we should now move this function in the http_client.h header so that we avoid duplicating code every-time we need this.